index.html

```diff
@@ -1,26 +1,113 @@
-<!DOCTYPE html> 
+<!DOCTYPE html>
 <html lang="en">
-  
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="device-width, initial-scale=1.0"> 
-    <link rel="stylesheet" href="styles.css"> 
-   </head>
-  
-  <body>
-    <header>Glazed Donuts</header> 
-    
-    <main> </main>
-    
-    
-    
-    
-    
-    
-    <!--https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fglazed%20donut01.jpg?v=1602793076372 --> 
-    <!--https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fglazed%20donut02.jpg?v=1602793077492 --> 
-    
-    
-  </body>
-  
-</html>
\ No newline at end of file
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+  <img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fglazed%20donut01.jpg?v=1602793076372" alt="">
+
+  <img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fspiceracklogo.png?v=1603324422595" alt="">
+
+  <h1>glazed Donuts</h1>
+
+  <ul>
+    <li><img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fcooktime.png?v=1603325721633" alt=""> 3
+      hr 30 min</li>
+    <li><img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fserves.png?v=1603325721686" alt="">14
+      donuts</li>
+    <li><img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fingredientss.png?v=1603325721633"
+        alt="">Click for ingredients</li>
+    <li><img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fcomments.png?v=1603325721633"
+        alt="">Comments</li>
```

Comment:
If and when the list item represents an actions the user can take to reveal more content, you should also add `a` elements.

Like so if you want the icon to _also_ be clickable:
```html
<li>
  <a href="#">
    <img src="your-image.png" alt=""> Click for ingredients
  </a>
</li>
```
Or like this if you want _only_ the text to be clickable:
```html
<li>
  <img src="your-image.png" alt="">
  <a href="#">Click for ingredients</a>
</li>
```